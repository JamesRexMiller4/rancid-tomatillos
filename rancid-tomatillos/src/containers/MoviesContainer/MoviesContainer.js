import React from 'react';
import MovieCard from '../../Components/MovieCard/MovieCard';
import { connect } from 'react-redux';
import './MoviesContainer.scss';
import { render } from 'react-dom';

const MoviesContainer = ({data}) => {
  const moviesList = data.movies.map(movie => {
    return (
      <MovieCard
        key={movie.id}
        id={movie.id}
        poster={movie.poster_path}
        avgRating={movie.average_rating}
      />
    )
  });

  return (
    <main className='main-movies-container'>
      {moviesList}
    </main>
  );
}

const mapStateToProps = state => ({
  data: state.movies
});

export default connect(mapStateToProps)(MoviesContainer);
