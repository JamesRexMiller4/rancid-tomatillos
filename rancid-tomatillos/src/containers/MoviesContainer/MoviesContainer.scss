.main-movies-container {
  display: flex;
  align-items: center;
  justify-content: space-around;
  flex-wrap: wrap;
  width: 100%;
  height: 70vh;
  overflow: scroll;
  background-color: #E0F3F4;
}
