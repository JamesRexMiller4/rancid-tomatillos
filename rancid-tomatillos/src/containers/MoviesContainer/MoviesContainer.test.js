import React from 'react';
import {shallow} from 'enzyme';
import MoviesContainer from './MoviesContainer';
 
describe('Movies Containers', () => {
let wrapper;
 
beforeEach(() => {
wrapper = shallow(<MoviesContainer />)
})
 
it.skip('', () => {
})
 
});