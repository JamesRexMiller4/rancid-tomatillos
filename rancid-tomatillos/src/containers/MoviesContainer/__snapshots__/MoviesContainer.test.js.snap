// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer MoviesContainer Component should match the snapshot with correct data passed in 1`] = `
<main
  className="main-movies-container"
>
  <MovieCard
    avgRating={6}
    id={1}
    isLoggedIn={true}
    key="1"
    poster="https://image.tmdb.org/t/p/original//l4iknLOenijaB85Zyb5SxH1gGz8.jpg"
    userRating={Array []}
  />
  <MovieCard
    avgRating={5.666666666666667}
    id={2}
    isLoggedIn={true}
    key="2"
    poster="https://image.tmdb.org/t/p/original//xBHvZcjRiWyobQ9kxBhO6B2dtRI.jpg"
    userRating={Array []}
  />
</main>
`;
